transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/SelectTransferMethodFragment.java

```diff
@@ -231,6 +234,12 @@ public void showTransferMethodCountry(@NonNull String countryCode) {
 
     @Override
     public void showTransferMethodTypes(@NonNull List<TransferMethodSelectionItem> transferMethodTypes) {
+        Map<String, String> params = new HashMap<>();
+        params.put("hyperwallet_ea_country", mSelectedCountryCode);
+        params.put("hyperwallet_ea_currency", mSelectedCurrencyCode);
```

Comment:
when core code is ready please use the following constants:
`InsightEventTag.InsightEventTagEventParams.TRANSFER_METHOD_COUNTRY`
`InsightEventTag.InsightEventTagEventParams.TRANSFER_METHOD_CURRENCY`